WordPress/src/main/java/org/wordpress/android/ui/reader/views/compose/tagsfeed/ReaderTagsFeed.kt

```diff
@@ -98,15 +101,17 @@ private fun Loaded(uiState: UiState.Loaded) {
             uiState.onRefresh()
         }
     )
-
     Box(
         modifier = Modifier
             .fillMaxSize()
             .pullRefresh(state = pullRefreshState),
     ) {
+        val listState = rememberLazyListState()
```

Comment:
💡  Since we're not using the state for anything, I believe we can keep using the default parameter in `LazyColumn` instead of explicitly defining the state.

Reply:
Fixed, thanks!